src/helperFunctions/install.py

```diff
@@ -179,8 +176,8 @@ def install_github_project(project_path: str, commands: List[str]):
 
 def _checkout_github_project(github_path: str, folder_name: str):
     clone_url = 'https://www.github.com/{}'.format(github_path)
-    _, return_code = execute_shell_command_get_return_code('git clone {}'.format(clone_url))
-    if return_code != 0:
+    git_p = subprocess.run('git clone {}'.format(clone_url), shell=True, stdout=DEVNULL, stderr=DEVNULL, text=True)
+    if git_p != 0:
```

Comment:
It seems `.returncode` is missing:
```suggestion
    if git_p.returncode != 0:
```